# Describes the mobility constraints of a camera mounted to a robot.
# Possible roll range is assumed 0.
# Possible yaw range is assumed 360 deg.

float64 height_min #[m]
float64 height_max #[m]
float64 pitch_min #[rad]
float64 pitch_max #[rad]
float64 hfov #[rad]
float64 vfov #[rad]
float64 range_min #[m]
float64 range_max #[m]
float64 roll #[rad]
